pkg/config/config_test.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric-protos-go/common"
+	. "github.com/onsi/gomega"
+)
+
+func TestSignConfigUpdate(t *testing.T) {
+	t.Run("success", func(t *testing.T) {
```

Comment:
Does this need to have a t.Run right here? It's the only test in this function

Reply:
agreed. when we implemented the test, we were thinking about adding more test cases in this function. it's definitely not necessary now.

Reply:
done